src/WebUI/Common/KubeSummary.scss

```diff
@@ -1,15 +1,33 @@
 .kubernetes-container {
     // cards have margins
-    .k8s-card-padding+.k8s-card-padding {
+    .k8s-card-padding + .k8s-card-padding {
         margin-top: 16px;
     }
-
     /*Css to adjust the alignment of text between picklist and text search. Revisit when we move from picklist to dropdown (fabric-free)*/
-    .keyword-search.bolt-text-field{
+    .keyword-search.bolt-text-field {
         padding-bottom: 8px;
     }
 
     .loading-pods {
         height: 200px;
     }
-}
\ No newline at end of file
+
+    .details-card-row-size {
+        padding-top: 6px;
+        padding-bottom: 6px;
+        display: flex;
+    }
+
+    .details-card-info-field-size {
+        margin-right: 8px;
+        width: 150px;
+        min-width: 150px;
+    }
```

Comment:
sorry to bother, as figma, 
min-width and width make it as 96px, margin-right make it as 20px; accordingly adjust the value field also.

also check the padding between title of the card and first row, as per figma it is 20px, means, 6px for padding and 14px for the otherone, same with the bottom padding also.

Reply:
Done. Have made changes to ensure the UI paddings/margins/fonts are same as in figma